src/FooParser.hs

```diff
@@ -5,8 +5,8 @@ import Text.Megaparsec
 import Text.Megaparsec.String -- input stream is of type ‘String’
 import qualified Text.Megaparsec.Lexer as L
 
-data Section = Section { username :: String
-                       , paragraphs :: [[String]]
+data Section = Section { sectionUsername :: String
```

Comment:
If the record is just sitting in its own module, we can pretend Haskell's records aren't so ugly and avoid prefixing.